convert2rhel/unit_tests/toolopts_test.py

```diff
@@ -20,17 +20,26 @@
 
 import sys
 import unittest
+import pytest
 
 import convert2rhel.toolopts
+import convert2rhel.utils
 
 from convert2rhel import unit_tests  # Imports unit_tests/__init__.py
 from convert2rhel.toolopts import tool_opts
 
 
-class TestToolopts(unittest.TestCase):
-    def _params(params):  # pylint: disable=E0213
+if sys.version_info[:2] <= (2, 7):
+    import mock  # pylint: disable=import-error
+else:
+    from unittest import mock  # pylint: disable=no-name-in-module
+
+
+def _params(params):  # pylint: disable=E0213
```

Comment:
pylint rule disabling can be removed, and naming could be improved. Suggestion could probably be described better too, append isn't entirely correct

```suggestion
def append_sys_params(params):
```